homeassistant/components/twinkly/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""The twinkly component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType
+
+
+async def async_setup(hass: HomeAssistantType, config: dict):
+    """Set up the twinkly integration."""
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistantType, config_entry: ConfigEntry):
+    """Set up entries from config flow."""
+
```

Comment:
I would recommend actually setting up the Twinkly client here instead of in the component itself. The reasoning behind this is for future extendability. Down the line it might be nice to add some services to set the "movie" mode on the lights or control them in some way that doesn't really fit the `light` platform. Take a look at the scaffolding example here: https://github.com/home-assistant/core/blob/dev/script/scaffold/templates/config_flow/integration/__init__.py#L26

Reply:
Good point, I've moved it in the __init__.py then 😊